pkg/validate/validate.go

```diff
@@ -122,7 +122,12 @@ func Files(fileNames []string, validationRules []*validationrule.ValidationRule,
 				var excludedRules []string
 				excludedRulesText, ok := originalRule.Annotations[excludeAnnotationName]
 				if ok {
-					excludedRules = strings.Split(excludedRulesText, ",")
```

Comment:
There doesn't seem to be a test file for this file, should I create one with test covering this case?

Reply:
Yes! It would be definitely welcomed if you add tests for parsing of the `excludedRulesText`!

feel free to move the code to a separate function so it can be easily tested  :+1: